pkg/minishift/systemtray/systemtray.go

```diff
@@ -0,0 +1,307 @@
+// +build !systemtray
+
+/*
+Copyright (C) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package systemtray
+
+import (
+	"fmt"
+	goos "os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"sync"
+	"time"
+
+	"github.com/anjannath/systray"
+	"github.com/docker/machine/libmachine"
+	libmachineState "github.com/docker/machine/libmachine/state"
+	"github.com/golang/glog"
+	"github.com/minishift/minishift/cmd/minishift/state"
+	"github.com/minishift/minishift/pkg/minikube/cluster"
+	"github.com/minishift/minishift/pkg/minishift/profile"
+	"github.com/minishift/minishift/pkg/minishift/systemtray/icon"
+	"github.com/minishift/minishift/pkg/util/os"
+	"github.com/minishift/minishift/pkg/util/slice"
+)
+
+const (
+	START = "Start"
+	STOP  = "Stop"
+	EXIT  = "Exit"
+
+	START_PROFILE  = 1
+	STOP_PROFILE   = 2
+	STOPPED        = 3
+	RUNNING        = 4
+	DOES_NOT_EXIST = 5
```

Comment:
@anjannath This should be `IOTA` type.

Reply:
should be? can be... it does not impeded the functionality or working of the code under review

Reply:
TBH i was using IOTA here, then the status was not getting reflected, so again moved to manually declaring the values, using IOTA make sense here.